export-osv.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/**
+ * Script for exporting advisories to OSV format.
+ *
+ * Usage: `php export-osv.php target_folder`
+ *
+ * @see https://ossf.github.io/osv-schema/
+ */
+
+namespace FriendsOfPhp\SecurityAdvisories;
+
+use DirectoryIterator;
+use FilesystemIterator;
+use SplFileInfo;
+use Symfony\Component\Yaml\Yaml;
+
+if (!is_file($autoloader = __DIR__ . '/vendor/autoload.php')) {
+    echo 'Dependencies are not installed, please run "composer install" first!' . PHP_EOL;
+    exit(1);
+}
+
+require $autoloader;
+
+function convertToOsv(SplFileInfo $fileInfo, string $package): array
+{
+    $advisory = Yaml::parseFile($fileInfo->getPathname());
+
+    return [
+        'id' => $advisory['cve'] ?? 'PHPSEC-' . $fileInfo->getBasename('.yaml'),
+        'modified' => getDateFromGitLog($fileInfo),
+        'published' => getDateFromGitLog($fileInfo, true),
+        'aliases' => [],
+        'related' => [],
+        'summary' => $advisory['title'] ?? '',
+        'details' => '',
+        'affected' => [
+            'package' => [
+                'ecosystem' => 'Packagist',
+                'name' => $package,
+                'purl' => sprintf('pkg:packagist/%s', $package),
+            ],
+            'ranges' => [
+                'type' => 'SEMVER',
+                'events' => getEvents($advisory['branches']),
+            ],
+        ],
+        'references' => [
+            array_key_exists('link', $advisory) ? [
+                'type' => 'ADVISORY',
+                'url' => $advisory['link'],
+            ] : null,
```

Comment:
putting `null` in the array looks wrong to me

Reply:
True. Since the validator enforces links anyway, I just skipped that check.